packages/teams-js/src/public/liveShare.ts

```diff
@@ -0,0 +1,404 @@
+     * These data objects and DDSes exist for the lifetime of the container.
+     */
+    readonly initialObjects: Record<string, unknown>;
+
+    /**
+     * The current attachment state of the container.  Once a container has been attached, it remains attached.
+     * When loading an existing container, it will already be attached.
+     */
+    readonly attachState: string;
+
+    /**
+     * A newly created container starts detached from the collaborative service.
+     * Calling `attach()` uploads the new container to the service and connects to the collaborative service.
+     *
+     * @remarks This should only be called when the container is in the
+     * {@link @fluidframework/container-definitions#AttachState.Detatched} state.
+     *
+     * This can be determined by observing {@link IFluidContainer.attachState}.
+     *
+     * @returns A promise which resolves when the attach is complete, with the string identifier of the container.
+     */
+    attach(): Promise<string>;
+
+    /**
+     * Attempts to connect the container to the delta stream and process operations.
+     * Will throw an error if unsuccessful.
+     *
+     * @remarks This should only be called when the container is in the
+     * {@link @fluidframework/container-definitions#ConnectionState.Disconnected} state.
+     *
+     * This can be determined by observing {@link IFluidContainer.connectionState}.
+     */
+    connect(): void;
+
+    /**
+     * Disconnects the container from the delta stream and stops processing operations.
+     *
+     * @remarks This should only be called when the container is in the
+     * {@link @fluidframework/container-definitions#ConnectionState.Connected} state.
+     *
+     * This can be determined by observing {@link IFluidContainer.connectionState}.
+     */
+    disconnect(): void;
+
+    /**
+     * Create a new data object or Distributed Data Store (DDS) of the specified type.
+     *
+     * @remarks In order to share the data object or DDS with other
+     * collaborators and retrieve it later, store its handle in a collection like a SharedDirectory from your
+     * initialObjects.
+     *
+     * @param objectClass - The class of data object or DDS to create
+     */
+    create<T>(objectClass: LoadableObjectClass<T>): Promise<T>;
+
+    /**
+     * Dispose of the container instance, permanently disabling it.
+     */
+    dispose(): void;
+  }
+
+  /**
+   * Initializes the Live Share client.
+   * @param options Optional. Configuration options passed to the Live Share client.
+   *
+   * @beta
+   */
+  export async function initialize(options?: LiveShareOptions): Promise<void> {
+    if (initializing || client) {
+      throw new Error('Live Share has already been initialized.');
+    }
+
+    try {
+      initializing = true;
+      const LiveShareClient = window['53de46f8-db62-4b8d-ae81-330f828ac86c'] as LiveShareClient;
```

Comment:
Please add a comment above the use of this guid explaining how it works and what it is a reference to.

Reply:
No problem

Reply:
Removed namespace per discussion